platform/kube/inject/testdata/first-initializer.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hello
+  namespace: default
+  initializers:
+    pending:
+    - name: sidecar.initializer.istio.io
+    - name: some.other.initializer
+spec:
+  replicas: 7
+  template:
+    spec:
+      containers:
+        - name: hello
+          image: "fake.docker.io/google-samples/hello-go-gke:1.0"
```

Comment:
How about adding imagePullPolicy here?

Reply:
I don't think it matters too much since this file is never posted to k8s apiserver.